### PR TITLE
upgrade arc perf test to transformers 4.37

### PR DESCRIPTION
**Pull request by cranechu0131**
upgrade arc perf test to transformers 4.37

### CODE REVIEW

File: .github/workflows/llm_performance_tests.yml [hunk 1]
```diff
@@ -153,7 +153,7 @@ jobs:
           source /opt/intel/oneapi/setvars.sh
           export USE_XETLA=OFF
           export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```
Comment (Oscilloscope98) [inline .github/workflows/llm_performance_tests.yml:155]: python -m pip install transformers==4.36.2

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -1519,5 +1519,4 @@ jobs:
       #   if: ${{ always() }}
       #   shell: cmd
       #   run: |
-      #     call conda env remove -n igpu-perf -y
-
```
Comment (Oscilloscope98) [inline .github/workflows/llm_performance_tests.yml:1523]: Please do not remove the empty line at the end of this file

**Review by Oscilloscope98 — COMMENTED** [review]
please also add `pip install transformers==4.37.0` for core and spr tests

**Review by Oscilloscope98 — COMMENTED** [review]
Others

**Review by Oscilloscope98 — APPROVED** [review]

**Review by hkvision — APPROVED** [review]

**Pull request merged**